account/src/main/java/com/user/account/advice/AllControllerExceptionHandler.java

```diff
@@ -0,0 +1,40 @@
+package com.user.account.advice;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.databind.exc.InvalidFormatException;
+import com.user.account.message.DefaultMessage;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import javax.validation.ConstraintViolationException;
+
+@ControllerAdvice
+public class AllControllerExceptionHandler {
+
+    @ExceptionHandler(value = NumberFormatException.class)
+    public ResponseEntity<String> amountNotNumber(){
+        return new ResponseEntity<>(DefaultMessage.TRANSACTION_FAILED_INVALID_AMOUNT, HttpStatus.BAD_REQUEST);
+    }
+
+    @ExceptionHandler(value = ConstraintViolationException.class)
+    public ResponseEntity<String> validationErrorFound(){
+        return new ResponseEntity<>(DefaultMessage.VALIDATION_FAILED, HttpStatus.BAD_REQUEST);
```

Comment:
`ConstraintViolationException` should not be 400, it can be 500 (`HttpStatus.INTERNAL_SERVER_ERROR`)

Reply:
Changed HttpStatus.BAD_REQUEST to HttpStatus.INTERNAL_SERVER_ERRO